creations/heltec.md

```diff
@@ -1,5 +1,8 @@
 # heltec-creations
 Community Allocated Creation IDs for Heltec boards
 
+## `0x0032_xxxx` - ESP32 boards
+*  `0x0032_0001` [WiFi Kit 32 (V2)](https://resource.heltec.cn/download/WiFi_Kit_32/WiFi%20Kit32.pdf)
```

Comment:
Let's make this 0002 since it's V2.
```suggestion
*  `0x0032_0002` [WiFi Kit 32 (V2)](https://resource.heltec.cn/download/WiFi_Kit_32/WiFi%20Kit32.pdf)
```